src/import/planner.c

```diff
@@ -970,13 +970,18 @@ replace_nestloop_params_mutator(Node *node, PlannerInfo *root)
 		/* Upper-level PlaceHolderVars should be long gone at this point */
 		Assert(phv->phlevelsup == 0);
 
+#if PG16_LT
 		/*
 		 * Check whether we need to replace the PHV.  We use bms_overlap as a
 		 * cheap/quick test to see if the PHV might be evaluated in the outer
 		 * rels, and then grab its PlaceHolderInfo to tell for sure.
 		 */
 		if (!bms_overlap(phv->phrels, root->curOuterRels) ||
 			!bms_is_subset(find_placeholder_info(root, phv, false)->ph_eval_at, root->curOuterRels))
+#else
```

Comment:
The find_placeholder_info now performs in constant time, so now we don't have to use bms_overlap as a initial test instead of directly calling find_placeholder_info. Relevant commit : postgres/postgres@6569ca439
 
 I have now updated the PR's commit message with this information.